frontend/src/pages/dashboard/components/sidebar/sidebar.tsx

```diff
@@ -39,7 +39,7 @@ const Sidebar: React.FC<Properties> = ({ isCollapsed = false }: Properties) => {
       <li
         className={getValidClassNames(
           styles.item,
-          checkActiveTab(location.pathname, tab.path) && styles.active,
+          location.pathname === tab.path && styles.active,
```

Comment:
why do you remove checkIsActiveTab?

Reply:
@kalinkaaaa14 It's my fault, the previous version of the helper did a `path === `${AppRoute.DASHBOARD}/${tab}`` comparison. Now it's just comparing incoming values without any additional logic over them. It seemed to me that this is a complication and the comparison can be done in place like we have in other parts of the application.

Reply:
@kalinkaaaa14 It's my fault, the previous version of the helper did a `path === ${AppRoute.DASHBOARD}/${tab}` comparison. Now it's just comparing incoming values without any additional logic over them. It seemed to me that this is a complication and the comparison can be done in place like we have in other parts of the application.

Reply:
it's ok, I'll change it